addons/background/package.json

```diff
@@ -30,6 +30,9 @@
     "prop-types": "^15.6.1",
     "react-lifecycles-compat": "^3.0.2"
   },
+  "devDependencies": {
+    "vue": "^2.5.16"
+  },
   "peerDependencies": {
     "react": "*"
```

Comment:
I think we should remove this peerDependency now (and suppress corresponding ESLint warnings)

Reply:
Do you mean the `react` or the `vue` dependency? or should I just remove both?

Reply:
or should I move the `react` dep into `devDependencies`

Reply:
I mean react. In devDependencies, you can have anything that's used in tests.

Reply:
Sounds good! Thanks for clarifying